<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
  <class>addrecord</class>
  <widget class="QMainWindow" name="addrecord">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>469</width>
        <height>487</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Add/Delete A Record</string>
    </property>
    <property name="windowIcon">
      <iconset>
        <normaloff>Resources/registro_icon_application.png</normaloff>Resources/registro_icon_application.png
      </iconset>
    </property>
    <widget class="QWidget" name="centralwidget">
      <widget class="QLabel" name="label">
        <property name="geometry">
          <rect>
            <x>160</x>
            <y>10</y>
            <width>111</width>
            <height>16</height>
          </rect>
        </property>
        <property name="font">
          <font>
            <pointsize>10</pointsize>
          </font>
        </property>
        <property name="text">
          <string>Add New Element</string>
        </property>
      </widget>
      <widget class="QLabel" name="label_2">
        <property name="geometry">
          <rect>
            <x>20</x>
            <y>40</y>
            <width>41</width>
            <height>16</height>
          </rect>
        </property>
        <property name="font">
          <font>
            <weight>75</weight>
            <bold>true</bold>
          </font>
        </property>
        <property name="text">
          <string>Mark:</string>
        </property>
      </widget>
      <widget class="QLabel" name="label_3">
        <property name="geometry">
          <rect>
            <x>180</x>
            <y>40</y>
            <width>61</width>
            <height>16</height>
          </rect>
        </property>
        <property name="font">
          <font>
            <weight>75</weight>
            <bold>true</bold>
          </font>
        </property>
        <property name="text">
          <string>Subject:</string>
        </property>
      </widget>
      <widget class="QDateEdit" name="dtVote">
        <property name="geometry">
          <rect>
            <x>310</x>
            <y>60</y>
            <width>110</width>
            <height>22</height>
          </rect>
        </property>
        <property name="calendarPopup">
          <bool>true</bool>
        </property>
        <property name="date">
          <date>
            <year>2000</year>
            <month>1</month>
            <day>2</day>
          </date>
        </property>
      </widget>
      <widget class="QComboBox" name="cbnSubject">
        <property name="geometry">
          <rect>
            <x>140</x>
            <y>60</y>
            <width>131</width>
            <height>22</height>
          </rect>
        </property>
      </widget>
      <widget class="QLabel" name="label_4">
        <property name="geometry">
          <rect>
            <x>340</x>
            <y>40</y>
            <width>47</width>
            <height>13</height>
          </rect>
        </property>
        <property name="font">
          <font>
            <weight>75</weight>
            <bold>true</bold>
          </font>
        </property>
        <property name="text">
          <string>Date:</string>
        </property>
      </widget>
      <widget class="QLineEdit" name="lnDesc">
        <property name="geometry">
          <rect>
            <x>40</x>
            <y>140</y>
            <width>381</width>
            <height>21</height>
          </rect>
        </property>
      </widget>
      <widget class="QLabel" name="label_5">
        <property name="geometry">
          <rect>
            <x>190</x>
            <y>120</y>
            <width>91</width>
            <height>16</height>
          </rect>
        </property>
        <property name="font">
          <font>
            <weight>75</weight>
            <bold>true</bold>
          </font>
        </property>
        <property name="text">
          <string>Description:</string>
        </property>
      </widget>
      <widget class="QPushButton" name="btnSaveVote">
        <property name="geometry">
          <rect>
            <x>10</x>
            <y>170</y>
            <width>441</width>
            <height>23</height>
          </rect>
        </property>
        <property name="text">
          <string>Save this vote</string>
        </property>
      </widget>
      <widget class="QLabel" name="lblDebug">
        <property name="geometry">
          <rect>
            <x>40</x>
            <y>420</y>
            <width>391</width>
            <height>20</height>
          </rect>
        </property>
        <property name="text">
          <string/>
        </property>
        <property name="alignment">
          <set>Qt::AlignCenter</set>
        </property>
      </widget>
      <widget class="QLabel" name="label_6">
        <property name="geometry">
          <rect>
            <x>170</x>
            <y>240</y>
            <width>111</width>
            <height>16</height>
          </rect>
        </property>
        <property name="font">
          <font>
            <pointsize>10</pointsize>
          </font>
        </property>
        <property name="text">
          <string>Delete An Element:</string>
        </property>
      </widget>
      <widget class="QLabel" name="label_7">
        <property name="geometry">
          <rect>
            <x>210</x>
            <y>320</y>
            <width>31</width>
            <height>16</height>
          </rect>
        </property>
        <property name="font">
          <font>
            <weight>75</weight>
            <bold>true</bold>
          </font>
        </property>
        <property name="text">
          <string>ID:</string>
        </property>
      </widget>
      <widget class="QPushButton" name="btnDelRecord">
        <property name="geometry">
          <rect>
            <x>10</x>
            <y>370</y>
            <width>431</width>
            <height>23</height>
          </rect>
        </property>
        <property name="text">
          <string>Delete By ID</string>
        </property>
      </widget>
      <widget class="QDoubleSpinBox" name="spnMark">
        <property name="geometry">
          <rect>
            <x>20</x>
            <y>60</y>
            <width>71</width>
            <height>29</height>
          </rect>
        </property>
        <property name="decimals">
          <number>2</number>
        </property>
        <property name="minimum">
          <double>2.000000000000000</double>
        </property>
        <property name="maximum">
          <double>10.000000000000000</double>
        </property>
      </widget>
      <widget class="QSpinBox" name="spnDelById">
        <property name="geometry">
          <rect>
            <x>200</x>
            <y>340</y>
            <width>49</width>
            <height>29</height>
          </rect>
        </property>
      </widget>
    </widget>
    <widget class="QMenuBar" name="menubar">
      <property name="geometry">
        <rect>
          <x>0</x>
          <y>0</y>
          <width>469</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QStatusBar" name="statusbar"/>
  </widget>
  <resources/>
  <connections/>
</ui>
